source/classes.tex

```diff
@@ -2303,9 +2303,9 @@
 \begin{note}
 A
 \tcode{return}
-statement\iref{stmt.return} in a destructor might not directly return to the
+statement\iref{stmt.return} in a destructor does not directly return to the
 caller; before transferring control to the caller, the destructors for the
-members and bases are called.
+members and bases (if any) are called.
```

Comment:
The pre-existing wording is unnecessarily verbose. I think the "does not" might be confusing in the case where there are no members or bases. How about replacing this with:

"A `return` statement ([stmt.return]) in a destructor calls the destructors for the members and bases (if any) before transferring control to the caller."

Reply:
fixed